config/locales/jobseekers.yml

```diff
@@ -130,11 +130,10 @@ en:
             opening: >-
               Add details of all jobs you’ve had since leaving full-time education. This should include:
             jobs_bullets:
-              - all teaching roles
-              - any other jobs you've had
+              - all paid roles
               - voluntary work, if relevant to your application
             closing: >-
-              You’ll also need to explain any gaps in your work history, including between finishing full-time education and starting work.
+              If you’re applying for a teaching job, you should include teaching and other experience, if you have it. You’ll also need to explain any gaps in your work history, including between finishing full-time education and starting work.
```

Comment:
Could these 2 sentences be in different paragraphs? (can't quite work out how to do that)

Reply:
still appearing as one block of text 😥 @ddippolito

Reply:
I believe this will require HTML changes and split the wording into two different translation values. As the whole `closing` one is [being included into a single HTML paragraph](https://github.com/DFE-Digital/teaching-vacancies/blob/1d09b64ed14985d592793089e78b3aae34c47f31/app/views/jobseekers/job_applications/build/employment_history.html.slim#L17)

Reply:
Think I address this one here
![Screenshot 2024-05-23 at 14 43 56](https://github.com/DFE-Digital/teaching-vacancies/assets/13124899/d84837ae-be6d-4342-b9d8-b889f501d0c5)